api/src/main/proto/stellarstation/api/v1/groundstation/groundstation.proto

```diff
@@ -119,16 +119,18 @@ message Plan {
   // need to configure transmission during the plan to match this device.
   radio.RadioDeviceConfiguration uplink_radio_device = 6;
 
-  // The start time of the plan.
+  // The start of the time window reserved for the plan.
```

Comment:
The comment should also refer that the time contains GS setup time and tear down time.